Scottish-Tech-Army/Volunteer-app
# Feature/sva 174

- [Description](#description)
  - [Type of change](#type-of-change)
- [Checklist](#checklist)


# Description

Allows the user to search for projects via free search:

-  The user can find this feature after tapping the search icon from the home screen
-  Typing and then confirming the search term will display all matching projects in a list
-  Excludes search through causes as there is no information pulled in for this yet
- Excludes use of redux

Ticket [SVA-174](https://sta2020.atlassian.net/browse/SVA-174)

## How to test

-  On the home screen, tab on the search icon
- Search for a term (for example "Volunteer App")
- Hit Enter
- Search results containing this term should be displayed

## Type of change



# Checklist
